sopt/src/main/scala/dagr/sopt/cmdline/ClpReflectiveBuilder.scala

```diff
@@ -252,8 +252,10 @@ private[sopt] class ClpArgument(declaringClass: Class[_],
     */
   private def prettyNameValue(value: Any): String = {
     Option(value) match {
-      case Some(v) if isSensitive => String.format("--%s ***********", longName)
-      case Some(v) => String.format("--%s %s", longName, v.toString)
+      case Some(v) if isSensitive  => String.format("--%s ***********", longName)
+      case Some(Some(optionValue)) => prettyNameValue(optionValue)
+      case Some(None)              => String.format("--%s %s", longName, ReflectionUtil.SpecialEmptyOrNoneToken)
+      case Some(v)                 => String.format("--%s %s", longName, v.toString)
       case _ => ""
```

Comment:
This makes me wonder if this shouldn't return `Option[String]` and then users can `flatMap` over it to eliminate missing options, vs. this will presumably introduce extra spaces into the final CLP string or require a `filterNot(_.isEmpty)`.